lawlibrary/views/home.py

```diff
@@ -9,6 +9,23 @@ class HomePageView(TemplateView):
 
     def get(self, request, *args, **kwargs):
         context = self.get_context_data()
+
```

Comment:
If you change this class to extend liiweb's HomePageView, then you can rely on that view to provide the courts.